lib/src/argument_list_visitor.dart

```diff
@@ -295,10 +294,11 @@ class ArgumentSublist {
   /// The full argument list from the AST.
   final List<Expression> _allArguments;
 
-  /// The positional arguments, in order.
+  /// The leading positional arguments, in order.
   final List<Expression> _positional;
 
-  /// The named arguments, in order.
+  /// The named arguments, in order, and any positional arguments that follow
+  /// at least one named argument.
```

Comment:
These comments don't match my understanding of `_splitArgumentLists`. IIUC This list can contain positional arguments that don't follow any named argument, as long as _some other_ positional argument follows some named argument.

Reply:
Good catch. Reworded.